app/boards/shields/nice_view_oled/boards/nice_nano.overlay

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright (c) 2022 The ZMK Contributors
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+nice_view_spi: &spi0 {
+	compatible = "nordic,nrf-spim";
+	status = "okay";
+	sck-pin = <20>;
+	mosi-pin = <17>;
+	miso-pin = <25>;
+	cs-gpios = <&pro_micro 1 GPIO_ACTIVE_HIGH>;
```

Comment:
I think omit this entirely? Have folks override the `cs-gpios` property in their keymap for whatever pin they bodged.

Reply:
It'd be nice to allow for this to be a default fallback. I'll likely put in the docs for the nice!view to use this pin if available. (it should be for most since it's on the trrs serial pin)

Reply:
As long as this is clearly documented there, and in the README, that's fine.